src/compass/utils/lut.py

```diff
@@ -362,3 +367,55 @@ def resample_set(geo_tide, pts_src, pts_dest):
                    bounds_error=False, fill_value=0)
     rdr_tide = rgi_func(pts_dest)
     return rdr_tide
+
+
+def ionosphere_delay(sensing_time, wavelength,
+                     ionex_path, lon_arr, lat_arr, inc_arr):
+    '''
+    Calculate ionosphere delay for geolocation
+
+    Parameters
+    ----------
+    time_sensing: datetime.datetime
+        Sensing time of burst
+    wavelength: float
+        Wavelength of the signal
+    lon_arr: numpy.ndarray
+        array of longitude in radar grid
+    lat_arr: numpy.ndarray
+        array of latitude in radar grid
+    inc_arr: numpy.ndarray
+        array of incidence angle in radar grid
+
+    Returns
+    -------
+    slant_range_delay: np.ndarray
+        Ionospheric delay in slant range
+    '''
+
+    if not ionex_path:
```

Comment:
It is ok to have this check inside the function but we need to discuss what is the default behavior. If a user wants to generate a CSLC-S1 does he/she need to always provide a TEC file?

Reply:
Let's resume the discussion on the default behavior in our next meeting. In the meantime, we do not want the user to get until here if the TEC file has not been provided. I recommend adding a check in `geo_runconfig.py` similarly to what has been done here https://github.com/vbrancat/COMPASS/blob/016eb4d762b440ba0609e9a8f5fd495a14e8f55a/src/compass/utils/geo_runconfig.py#L73 so that we stop the SAS immediately if the TEC file is not provided

Reply:
With respect to the SAS default behavior, just a heads up, I remember we decided to support an interface where the provided path to the TEC file might be `None`

Reply:
@vbrancat  Thanks for the suggestion and information. I've added the check in `geo_runconfig.py` for TEC file. Also, in `iono.ionosphere_delay()`, I've modified it to return a zero-filled array when `tec_file` is `None`

```python
if not tec_file:
    print('"tec_file" was not provided. '
             'Ionosphere correction will not be applied.')
    return np.zeros(lon_arr.shape)
```